atomics/used_guids.txt

```diff
@@ -665,3 +665,4 @@ ab042179-c0c5-402f-9bc8-42741f5ce359
 9c6bdb34-a89f-4b90-acb1-5970614c711b
 127b4afe-2346-4192-815c-69042bec570e
 9f5d081a-ee5a-42f9-a04e-b7bdc487e676
+42d33451-0a3c-4292-b753-9dbdcd67a0d3
```

Comment:
Please remove the change here so that the build system can add the auto_generated_guid during merging of the PR. This is the best way to avoid merge conflicts, thank you.